### ISSUE TITLE
DateTimeOffset serialization is culture dependent in 2.0.0-preview

### ISSUE THREAD

**Issue by gdeOo** [microsoftgraph/msgraph-sdk-dotnet-core#232]
Please provide the following (and please check them off the list with [x]) before submitting this issue:
- [x] Expected behavior. Please provide **links to the specific [Microsoft Graph documentation](https://developer.microsoft.com/en-us/graph/docs/concepts/overview)** you used to determine the expected behavior. 
- [x] Actual behavior. Provide error codes, stack information, and a [Fiddler](http://www.telerik.com/fiddler) capture of the request and response (please remove personally identifiable information before posting).
- [x] Steps to reproduce the behavior. Include your code, IDE versions, client library versions, and any other information that might be helpful to understand your scenario.

### Expected behavior

When creating a `Subscription`, which includes the required `ExpirationDateTime` fields of type `DateTimeOffset`, I expected the following field assignment to work based on the [examples in the documentation](https://docs.microsoft.com/en-us/graph/api/subscription-post-subscriptions?view=graph-rest-1.0&tabs=csharp) and elsewhere:

```c#
await _client.Subscriptions.Request()
    .AddAsync(new Subscription
    {
        ExpirationDateTime = DateTimeOffset.UtcNow.AddMinutes(5),
        ...
    });
```

### Actual behavior

The request fails with the following message: `Could not process subscription creation payload. Are all property names spelled and camelCased properly?Also are the dateTimeOffest properties in a valid internet Date and Time format?`.

Looking at the request that gets sent to the API, I noticed that the `expirationDateTime` field is formatted according to the current culture instead of using the format the API expects:

```json
{
  "expirationDateTime": "25/03/2021 18:10:53 +00:00",
  ...
}
```

After a bit of debugging, I believe the culprit is the `DateTimeOffsetConverter` used [here](https://github.com/microsoftgraph/msgraph-sdk-dotnet-core/blame/Microsoft.Graph.Core.2.0.0-preview.8.nupkg/src/Microsoft.Graph.Core/Serialization/Serializer.cs#L40), as it converts the timestamp to string [using `ToString`](https://github.com/microsoftgraph/msgraph-sdk-dotnet-core/blob/Microsoft.Graph.Core.2.0.0-preview.8.nupkg/src/Microsoft.Graph.Core/Serialization/DateTimeOffsetConverter.cs#L51), whose output is culture dependent.

### Steps to reproduce the behavior

Create a `Subscription` while using `Microsoft.Graph.Core (2.0.0-preview.8)` through `Microsoft.Graph.Beta (4.0.0-preview)`.

 
[AB#8745](https://microsoftgraph.visualstudio.com/0985d294-5762-4bc2-a565-161ef349ca3e/_workitems/edit/8745)

**Comment by andrueastman** [comment]
Hey @gdeOo!
Thanks for raising this issue and trying out our new preview!
We will resolve this in the next update.

**Comment by andrueastman** [comment]
Closed via #234
